exchanges/okx/okx_wrapper.go

```diff
@@ -780,7 +780,11 @@ func (ok *Okx) SubmitOrder(ctx context.Context, s *order.Submit) (*order.SubmitR
 		if s.Type.Lower() == "" {
 			orderRequest.OrderType = OkxOrderOptimalLimitIOC // only applicable for Futures and Perpetual Swap Types.
 		}
-		orderRequest.PositionSide = positionSideLong
+		if s.Side.IsLong() {
```

Comment:
Please add a `// TODO: handle positionSideLong while side is Short and positionSideShort while side is Long`. GCT doesn't currently have the ability to handle buying a short in the way OKx describes